core/modules/replica/ControllerApp.cc

```diff
@@ -381,318 +375,245 @@ ControllerApp::ControllerApp(int argc, char* argv[])
 
 int ControllerApp::runImpl() {
 
-    auto const controller = Controller::create(serviceProvider());
+    string const context = "ControllerApp::" + string(__func__) + " ";
 
-    atomic<bool> finished(false);
-    Request::Ptr request;
+    auto const controller = Controller::create(serviceProvider());
+    Request::Ptr ptr;
 
     if ("REPLICATE" == _request) {
-        request = controller->replicate(
+        ptr = controller->replicate(
             _workerName,
             _sourceWorkerName,
             _databaseName,
             _chunkNumber,
-            [&finished] (ReplicationRequest::Ptr const& request) {
-                ::printRequest<ReplicationRequest>(request);
-                finished = true;
+            [] (ReplicationRequest::Ptr const& ptr_) {
+                ::printRequest<ReplicationRequest>(ptr_);
```

Comment:
I think `request` was easier to read than `ptr_`

Reply:
The previous version of the code was saying "request" way too much.